Sources/BraintreeApplePay/BTApplePayClient.swift

```diff
@@ -115,11 +110,30 @@ import BraintreeCore
         ]
     }
 
-    func completionHandler(
-        onMainThreadWithPaymentRequest paymentRequest: PKPaymentRequest?,
-        error: Error?,
+    // MARK: - Analytics Completion Helpers
+
+    enum AnalyticsResult {
+        case success
+        case failure
+    }
```

Comment:
I know at one point support and API support used the entire `ios:event-name:event-type` string while searching for solutions to merchant inbounds. This may no longer be relevant, but something to consider. An enum will be less error prone and having a core type shared across each module will make the code more DRY.

Reply:
Yeah, when this feature branch is merged (or about to be merged) we will need to let Sales and Support know about this change, both in formatting and simplifying the events. Overall it should tell a more holistic story of a merchants conversion rate vs now is much harder to reason about.

Agree on the `enum` being less error prone, I'll make that change and if folks disagree we can move it back out.